scripts/debug_vt100_input.py

```diff
@@ -1,21 +1,14 @@
 #!/usr/bin/env python
 """Parse vt100 input and print keys."""
 
-from __future__ import annotations
-
 import sys
-from typing import TYPE_CHECKING
-
 from prompt_toolkit.input.vt100 import raw_mode
 
 from euporie.core.io import Vt100Parser
 from euporie.core.keys import Keys
 
-if TYPE_CHECKING:
-    from prompt_toolkit.key_binding import KeyPress
-
 
-def callback(key_press: KeyPress) -> None:
```

Comment:
I have chosen to use type annotations in this code-base. because I find it helps me catch errors and bugs.

Why do you think they should be removed from this script in particular?